redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
 Please refer to the <<ReleaseNotesRedfish>> section for further information
 about the power management actions and statuses.
 
+=== Physical Server Provisioning

```

Comment:
Perhaps a note would be in order to warn user she needs to have PXE server configured upfront and that PXE configuration is not really part of this documentation.

Reply:
👍 
Warning them implicitly.